### DISCUSSION TITLE
피드에 업로드시에 함께한 유저를 추가하는 기능이 가능할까요?

### DISCUSSION THREAD

**Discussion by ted002** [naverz/zepeto-studio-kor#482] [5. Q&A]
현재 피드에 업로드하면 플레이어만 나오고 추가적으로 함께한 유저를 추가하는 기능은 없는것으로 알고있습니다.
해당 기능을 추가하기에는 어려울까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 말씀하신대로 현재는 해당 기능이 없으며 
당장은 개발 중인 다른 기능들이 많아서 계획에 없는 부분 양해 부탁드립니다.
감사합니다.
